ProjectLighthouse.Servers.Website/Startup/WebsiteStartup.cs

```diff
@@ -44,6 +47,20 @@ public void ConfigureServices(IServiceCollection services)
 
         services.AddDbContext<DatabaseContext>();
 
+        services.AddHttpClient<ICaptchaService, CaptchaService>("CaptchaAPI",
+            client =>
+            {
+                Uri captchaUri = ServerConfiguration.Instance.Captcha.Type switch
+                {
+                    CaptchaType.HCaptcha => new Uri("https://hcaptcha.com"),
```

Comment:
Is the hCaptcha API root URL correct? I don't use hCaptcha but I just wanted to make sure.

Reply:
Just looked into the docs a bit, seems OK.